style.css

```diff
@@ -63,6 +95,229 @@ h1 {
   border: 2px solid #dfe1e6;
 }
 
+#work-section h2 {
+  font-size: 40px;
+  text-align: center;
+  color: #172b4d;
+  padding: 30px;
+}
+
+#card-container {
+  width: 100%;
+  display: grid;
+  justify-content: center;
+  padding: 10px;
+  gap: 15px;
+  grid-template-columns: repeat(auto-fill, 327px);
+}
+
+#about-me-text {
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 15-02-28](https://user-images.githubusercontent.com/66476329/236190216-b97250e0-e364-493c-a77b-f0209adae189.png)

</td>
<td>

![Screenshot from 2023-05-04 15-00-51](https://user-images.githubusercontent.com/66476329/236190243-25cb8b89-95aa-41c2-9de4-8bda41e7a9f8.png)

</td>
</tr>
</table>